src/libs/SidebarUtils.js

```diff
@@ -360,7 +371,7 @@ function getOptionData(report, reportActions, personalDetails, preferredLocale,
             result.alternateText = lastAction && lastMessageTextFromReport.length > 0 ? lastMessageText : Localize.translate(preferredLocale, 'report.noActivityYet');
         }
     } else {
-        if (!lastMessageText) {
+        if (!lastMessageText || ((result.isExpenseRequest || result.isIOURequest) && isDeletedParentAction)) {
```

Comment:
`(result.isExpenseRequest || result.isIOURequest)` could be replaced with `ReportUtils.isMoneyRequest`.